app/controllers/items_controller.rb

```diff
@@ -29,14 +29,17 @@ def index                                   #items in library
 
   def admin_index                             #denied and pending items in library
     @items = search
-
-    @pendings, @denieds = [], []
+    
+    @pendings, @denieds, @approveds = [], [], []
 
     @items.each do |item|
       @pendings << item if item.status == Item::PENDING
       @denieds << item if item.status == Item::DENIED
+      @approveds << item if item.status == Item::APPROVED
```

Comment:
Could you spend some time digging into why `@items.where(status: Item::APPROVED)` doesn't work here?

Reply:
Hi @spinecone, I fixed it. Turnout, it is because I used || instead of built-in or() to create the union of the search result :). I was a bit more rookie weeks ago.